.circleci/config.yml

```diff
@@ -124,7 +124,9 @@ jobs:
             echo $ENCODED_GOOGLE_CREDENTIALS | base64 -d > $GOOGLE_APPLICATION_CREDENTIALS
             echo "export GCLOUD_SERVICE_KEY=\$(echo \$ENCODED_GOOGLE_CREDENTIALS | base64 --decode)" >> $BASH_ENV
       - gcp-gcr/gcr-auth
-      - run: sudo -E .circleci/build_and_push_image.sh
+      - run: |
+          sudo chmod -R 777 /home/circleci/.docker && \
```

Comment:
YOLO. Seems fine for CI, but just wondering if a lighter touch like  `chown -R circleci:circleci /home/circleci/.docker` (make sure the files are owned by user)  or `chmod -R o+r /home/circleci/.docker` would work. Could you mention in the PR description what the issues is? Are there some files owned by root?

Reply:
Lighter touch worked 😅.